packages/core/src/browser/menu/browser-menu-plugin.ts

```diff
@@ -96,6 +96,7 @@ class DynamicMenuBarWidget extends MenuBarWidget {
             super['_openChildMenu']();
         };
     }
+
```

Comment:
empty line

Reply:
Do we have agreed on such formatting? I do like it better with an empty line (not that I really want to get into a discussion about it :-))

Reply:
We have not discussed it I assumed it from the rest of the code and I also prefer that...
It looked like a left over to me since no other code was touched in that function.

Reply:
It actually was a formatting change, because I like that empty line :)

Reply:
We can maybe discuss how to go about such things in today's meeting.